src/libmongoc/tests/unified/runner.c

```diff
@@ -738,32 +731,6 @@ check_run_on_requirement(test_runner_t *test_runner,
          continue;
       }
 
-      if (0 == strcmp(key, "serverless")) {
```

Comment:
Suggest checking and ignoring a `serverless` `runOnRequirement`:
```c
if (0 == strcmp(key, "serverless")) {
   // Ignore deprecated serverless runOnRequirement. Serverless is not tested.
   continue;
}
```

This avoids the test runner error `Unexpected runOnRequirement field: serverless` if running a test that includes a now-unused `serverless` `runOnRequirement`.